docker-compose.yml

```diff
@@ -17,11 +17,12 @@ services:
       - PRODUCTION=true
       - PROCESSES=4
       - THREADS=10
+    user: "1000:10001"
```

Comment:
Explicitly declare the user security context for readability. It's the same value with the recipe.